app/Http/Controllers/BuildMetricController.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\BuildMetric;
+use App\Character;
+use App\Gun;
+use App\Overclock;
+use Artesaos\SEOTools\Facades\SEOTools;
+use Illuminate\Http\Request;
+
+class BuildMetricController extends Controller
+{
+    public function index(Request $request, $class, $gun, $combo)
+    {
+        // SEOTools::setTitle('Deep Rock Galactic Loadouts');
+        // TODO: SEO
+
+        $build = BuildMetric::where([
+            ['character_id', '=', $class],
+            ['gun_id', '=', $gun],
+            ['build_combination', '=', $combo],
+        ])->get();
+
+        $build_gun = Gun::where('id', $gun)->get();
+        $gun_icon = asset('/assets/'.$build_gun[0]->image.'.svg');
```

Comment:
This should get moved to a helper on the Gun Model I think.

Reply:
No longer needed, due to the way we look up the icon and how the icon name is stored. Helper not needed.